src/OpenTelemetry.ResourceDetectors.Azure/AppServiceResourceDetector.cs

```diff
@@ -24,6 +24,7 @@ namespace OpenTelemetry.ResourceDetectors.Azure;
 /// <summary>
 /// Resource detector for Azure AppService environment.
 /// </summary>
+[Obsolete("Use AzureResourceDetector instead")]
```

Comment:
this package is not stable, so no need to obselete. we can just remove things.. assuming this is the direction agreed by the owners.